### DISCUSSION TITLE
ライセンスについて

### DISCUSSION THREAD

**Discussion by tkoyama010** [FEABeRjp/FEABeR#11] [General]
ライセンスはクリエイティブ・コモンズ 表示 4.0 国際 ライセンスを使用します。ご意見があればお願いいたします。
<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="クリエイティブ・コモンズ・ライセンス" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />この 作品 は <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">クリエイティブ・コモンズ 表示 4.0 国際 ライセンス</a>の下に提供されています。

**Comment by mmer547** [comment]
CC BYでいいと思います。

**Reply by tkoyama010** [reply]
対応しました。https://github.com/FEABeRjp/FEABeR/pull/13
